### PR TITLE
Use Admiral cert for Harbor

### LINKED ISSUE #531: The installer has to import the Harbor SSL certificate into the Admiral trust store as part of the post installation procedure
Admiral has to accept the SSL certificate from Harbor when Harbor is configured to communicate over HTTPS.
The OVA has to import the Harbor SSL certificate into the Admiral trust store as part of the post installation procedure and Admiral will automatically accept it.

### PR DESCRIPTION
**Pull request by andrewtchin**
Fixes #531

### CODE REVIEW

**Comment by andrewtchin** [comment]
@martin-borisov @lazarin @reasonerjt Please review this

File: installer/packer/scripts/harbor/configure_harbor.sh [hunk 1]
```diff
@@ -18,23 +18,17 @@ umask 077
...
-script_dir="/etc/vmware"
 harbor_compose_file="${conf_dir}/docker-compose.yml"
 
-cert_dir="${data_dir}/cert"
-flag="${conf_dir}/cert_gen_type"
 cfg="${data_dir}/harbor.cfg"
 
 ca_download_dir="${data_dir}/ca_download"
-mkdir -p "${cert_dir}"
+rm -rf "${ca_download_dir}"
```
Comment (reasonerjt) [inline installer/packer/scripts/harbor/configure_harbor.sh]: the same {ca_download_dir} is removed and created again in line 39,40.

File: installer/packer/scripts/harbor/configure_harbor.sh [hunk 2]
```diff
@@ -18,23 +18,17 @@ umask 077
...
 data_dir="/data/harbor"
 conf_dir="/etc/vmware/harbor"
-script_dir="/etc/vmware"
 harbor_compose_file="${conf_dir}/docker-compose.yml"
 
-cert_dir="${data_dir}/cert"
-flag="${conf_dir}/cert_gen_type"
 cfg="${data_dir}/harbor.cfg"
 
 ca_download_dir="${data_dir}/ca_download"
```
Comment (reasonerjt) [inline installer/packer/scripts/harbor/configure_harbor.sh]: When the certs are auto generated by .ova, the CA cert should be copied to this directory so Admin can download the CA cert from UI.

File: installer/packer/scripts/harbor/configure_harbor.sh (hunk 1 above)
Comment (andrewtchin) [inline installer/packer/scripts/harbor/configure_harbor.sh]: fixed

File: installer/packer/scripts/harbor/configure_harbor.sh (hunk 2 above)
Comment (andrewtchin) [inline installer/packer/scripts/harbor/configure_harbor.sh]: fixed

**Comment by andrewtchin** [comment]
@sergiosagu Please review this one since Lazarin is gone

**Pull request merged**